wp-content/plugins/zippicks-geo-service/includes/class-installer.php

```
@@ -0,0 +1,401 @@
+        // Update database version
+        update_option('zippicks_geo_db_version', self::DB_VERSION);
+        
+        // Add geohash column to restaurants table if it exists
+        self::update_restaurants_table();
+    }
+    
+    /**
+     * Update restaurants table with geo columns
+     */
+    private static function update_restaurants_table() {
+        global $wpdb;
```

PHPMD (2.15.0)

Avoid unused local variables such as '$wpdb'. (Unused Code Rules)

(UnusedLocalVariable)